python/taichi/lang/expr.py

```diff
@@ -315,10 +315,13 @@ def fill(self, val):
         fill_tensor(self, val)
 
     def __rpow__(self, power, modulo=None):
+        # Python will try Matrix.__pow__ first so no worry
```

Comment:
Could you be more specific here: what are we worrying about exactly? :-)

Reply:
e.g. What if we call `Matrix ** Expr`? Will it call `Expr.__rpow__` or `Matrix.__pow__`? We know it's `Matrix.__pow__`, so we don't need to add an `if ti.is_taichi_class...` guard like L323 does.